javascript/Round.js

```diff
@@ -11,18 +10,19 @@ class Round {
 
 // createRound();
 
-createRound() {
-  const categories = this.createCategories();
-  const categoryIds = this.getCategoryIds(categories);
-}
+  createRound() {
+    const categories = this.createCategories();
+    const categoryIds = this.getCategoryIds(categories);
+  }
```

Comment:
Both variables in this method are also not accessible outside of `createRound` -- do you need to reference them elsewhere? If so, you might want to set them as instance variables with `this.` instead.